vendor/bat-native-ads/src/bat/ads/internal/account/confirmations/confirmations_state.cc

```diff
@@ -676,6 +676,14 @@ bool ConfirmationsState::ParseAdRewardsFromDictionary(
     return false;
   }
 
+  const base::Value* unblinded_tokens =
+      dictionary->FindListKey("unblinded_payment_tokens");
+  if (unblinded_tokens && unblinded_tokens->GetList().empty()) {
+    // Migration path for https://github.com/brave/brave-browser/issues/16678
+    ad_rewards_dictionary->SetDoubleKey(
+        "unreconciled_estimated_pending_rewards", 0.0);
+  }
```

Comment:
Migration path to set `unreconciled_estimated_pending_rewards` to `0.0` if the user has no unreconciled unblinded payment tokens